pkg/ovsdb/ovsdb.go

```diff
@@ -30,26 +32,85 @@ const (
 	ovsTable    = "Open_vSwitch"
 )
 
-// Bridge Model of the Bridge table
+type (
+	// BridgeFailMode defines failure mode
+	BridgeFailMode = string
+	// BridgeProtocols defines OpenFlow Protocol version
+	BridgeProtocols = string
+)
+
+const (
+	// BridgeFailModeStandalone defines standalone mode
+	BridgeFailModeStandalone BridgeFailMode = "standalone"
+	// BridgeFailModeSecure defines secure mode
+	BridgeFailModeSecure BridgeFailMode = "secure"
+	// BridgeProtocolsOpenflow10 defines OpenFlow version 1.0
+	BridgeProtocolsOpenflow10 BridgeProtocols = "OpenFlow10"
+	// BridgeProtocolsOpenflow11 defines OpenFlow version 1.1
+	BridgeProtocolsOpenflow11 BridgeProtocols = "OpenFlow11"
+	// BridgeProtocolsOpenflow12 defines OpenFlow version 1.2
+	BridgeProtocolsOpenflow12 BridgeProtocols = "OpenFlow12"
+	// BridgeProtocolsOpenflow13 defines OpenFlow version 1.3
+	BridgeProtocolsOpenflow13 BridgeProtocols = "OpenFlow13"
+	// BridgeProtocolsOpenflow14 defines OpenFlow version 1.4
+	BridgeProtocolsOpenflow14 BridgeProtocols = "OpenFlow14"
+	// BridgeProtocolsOpenflow15 defines OpenFlow version 1.5
+	BridgeProtocolsOpenflow15 BridgeProtocols = "OpenFlow15"
+)
+
+// Bridge defines an object in Bridge table
 type Bridge struct {
-	UUID        string            `ovs:"_uuid"`
-	Name        string            `ovs:"name"`
-	OtherConfig map[string]string `ovs:"other_config"`
-	ExternalIds map[string]string `ovs:"external_ids"`
-	Ports       []string          `ovs:"ports"`
-	Status      map[string]string `ovs:"status"`
+	UUID                string            `ovsdb:"_uuid"`
```

Comment:
We don't  use most of these. Could we clean it up, so we don't bloat the module too much? This project has the benefit of not being a library but a focused application. We should only keep code that is needed for the project.

Reply:
Ok, fixed.